jobbergate-api/Dockerfile-dev

```
@@ -1,28 +1,19 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM python:3.8-slim-buster
 
-RUN apt-get -y update
-RUN apt-get install -y --fix-missing build-essential cmake libpq-dev curl
+RUN apt update && apt install -y curl libpq-dev gcc
 
-RUN pip install 'poetry==1.1.7'
+RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/poetry python && \
```

I usually fix the poetry version as well, so you can guarantee it will work as you think it will

```suggestion
RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/poetry python POETRY_VERSION=1.1.13 && \
```

---

+1 for that

---

Resolved in commit 3d7b178